index.html

```diff
@@ -5,18 +5,20 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/main.css" />
+    <script src="./js/script.js" defer></script>
+    <script src="https://kit.fontawesome.com/550c21f898.js" crossorigin="anonymous"></script>
     <link rel="icon" href="./favicon.ico" />
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
+        <i class="fa-bars fas"></i>
       </button>
       <nav id="desktop-nav">
         <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#projects">Portfolio</a></li>
           <li><a class="nav-link" href="#about-me">About</a></li>
           <li>
             <a class="nav-link" href="#contact"
```

Comment:
I would kindly suggest that you use an indentation of 2 spaces instead of 4, that is a rule that every day more will be asked of you